### PR TITLE
Update Arabic (ar) Homepage and Essentials buckets

### PR DESCRIPTION
**Pull request by minimalsm**
## Description

Update Arabic (ar) Homepage and Essentials buckets

### CODE REVIEW

File: src/intl/ar/page-what-is-ethereum.json [hunk 1]
```diff
@@ -1,69 +1,100 @@
...
+  "page-what-is-ethereum-slide-1-title": "المدفوعات عبر الحدود أرخص وأسرع",
+  "page-what-is-ethereum-slide-1-desc-1": "<b>العملات التابعة</b> هي نوع جديد من العملات الرقمية التي تعتمد على أصل أكثر استقرارًا كأساس لقيمتها. ويرتبط معظمها بدولار الولايات المتحدة ثَمّ تحافظ على قيمة تلك العملة. وتسمح لنظام دفع عالمي رخيص للغاية ومستقر. تم بناء العديد من العملات التابعة الحالية على شبكة الإيثيريوم.",
+  "page-what-is-ethereum-slide-1-desc-2": "وتبسِّط الإيثيريوم والعملات التابعة عملية إرسال الأموال إلى الخارج. وكثيرًا ما يستغرق نقل الأموال عبر العالم بضع دقائق فحسب. على عكس عدد أيام العمل أو حتى الأسابيع التي قد يستغرقها البنك العادي، ولجزء من السعر. وبالإضافة إلى ذلك، لا يوجد رسم إضافي لإجراء معاملة عالية القيمة، وهناك قيود صفرية على مكان أو سبب إرسال أموالك.",
+  "page-what-is-ethereum-slide-2-title": "مساعدة أسرع في أوقات الأزمات",
+  "page-what-is-ethereum-slide-2-desc-1": "إذا كنت محظوظًا بما يكفي لامتلاكك خيارات مصرفية متعددة من طريق مؤسسات موثوقة حيث تعيش، فقد تعتبر الحرية المالية والأمان والاستقرار أمر مسلم به. لكن بالنسبة للعديد من الأشخاص حول العالم الذين يواجهون القمع السياسي أو الصعاب الاقتصادية، قد لا توفر المؤسسات المالية الحماية أو الخِدْمَات.",
```
Comment (minimalsm) [inline src/intl/ar/page-what-is-ethereum.json:56]: Fix this

File: src/intl/ar/page-wallets-find-wallet.json [hunk 2]
```diff
@@ -106,7 +106,7 @@
   "page-find-wallet-new-to-wallets": "جديد في المحافظ؟  إليك نظرة عامة لتبدأ.",
   "page-find-wallet-new-to-wallets-link": "محافظ إثيريوم",
   "page-find-wallet-not-all-features": "لا توجد محفظة لديها كل هذه الميزات",
-  "page-find-wallet-not-endorsements": "المحافظ المدرجة في هذه الصفحة ليست موافقات رسمية ، ويتم توفيرها للأغراض الإعلامية فقط.  تم توفير أوصافهم من قبل شركات المحفظة نفسها.  نضيف منتجات إلى هذه الصفحة بناءً على المعايير الموجودة لدينا",
+  "page-find-wallet-not-endorsements": "Wallets listed on this page are not official endorsements, and are provided for informational purposes only. Their descriptions have been provided by the wallet projects themselves. We add products to this page based on criteria in our",
```
Comment (minimalsm) [inline src/intl/ar/page-wallets-find-wallet.json:109]: Regression

File: src/intl/ar/page-wallets-find-wallet.json [hunk 3]
```diff
@@ -63,7 +63,7 @@
   "page-find-wallet-description-rainbow": "منزل أفضل لأصول إثيريوم الخاصة بك",
   "page-find-wallet-description-samsung": "حافظ على الأشياء الثمينة الخاصة بك آمنة ومأمونة مع سلسلة كتل Samsung.",
   "page-find-wallet-description-status": "تطبيق المراسلة الآمنة ومحفظة التشفير ومتصفح Web3 المصمم بأحدث التقنيات",
-  "page-find-wallet-description-tokenpocket": "TokenPocket：محفظة عملات رقمية آمنة ومريحة ورائدة عالميًا وبوابة للتطبيقات اللامركزية، مع دعم سلاسل متعددة.",
+  "page-find-wallet-description-tokenpocket": "TokenPocket：A secure and convenient world-leading digital currency wallet and a portal to dapps, with Multi-chain supported.",
```
Comment (minimalsm) [inline src/intl/ar/page-wallets-find-wallet.json:66]: Regression

File: src/intl/ar/page-wallets-find-wallet.json [hunk 4]
```diff
@@ -39,19 +39,19 @@
...
   "page-find-wallet-description-keystone": "محفظة Keystone هي محفظة أجهزة عبر الفجوة الهوائية بنسبة 100% مع برنامج ثابت مفتوح المصدر وتستخدم بروتوكول رمز QR.",
   "page-find-wallet-description-ledger": "حافظ على الأصول الخاصة بك آمنة بأعلى معايير الأمان",
   "page-find-wallet-description-linen": "محفظة العقد الذكية للجوّال: اربح عائدًا، اشترِ العملات المشفرة، وشارك في DeFi. كسب المكافآت ورموز الحوكمة.",
   "page-find-wallet-description-loopring": "أول محفظة عقد ذكي من نوعها على الإطلاق لإيثيريوم ذات إمكانية التداول باستخدام \"zkRollup\"، والتحويلات، و \"AMM\". تتميّز بكونها خالية من الغاز وآمنة وبسيطة.",
-  "page-find-wallet-description-mathwallet": "MathWallet عبارة عن محفظة تشفير عالمية متعددة المنصات (للجوال / الامتداد / الويب) تدعم أكثر من 50 سلسلة كتل و2000+ تطبيق لامركزي.",
-  "page-find-wallet-description-metamask": "ابدأ في استكشاف تطبيقات سلسلة الكتل في ثوانٍ. موثوق به من قبل أكثر من مليون مستخدم حول العالم.",
+  "page-find-wallet-description-mathwallet": "MathWallet is a multi-platform (mobile/extension/web) universal crypto wallet that supports 50+ blockchains and 2000+ dapps.",
```
Comment (minimalsm) [inline src/intl/ar/page-wallets-find-wallet.json:53]: Regression

File: src/intl/ar/page-wallets-find-wallet.json [hunk 5]
```diff
@@ -39,19 +39,19 @@
   "page-find-wallet-description-coinbase": "التطبيق الآمن لتخزين العملات المشفرة بنفسك",
   "page-find-wallet-description-coinomi": "Coinomi هي أقدم محفظة متعددة السلاسل وجاهزة للتحدي ومتعددة المنصات لعملة البيتكوين والعملات البديلة والرموز - لم يتم اختراقها مطلقًا، مع ملايين المستخدمين.",
   "page-find-wallet-description-coin98": "محفظة غير إيداع متعددة السلاسل وبوابة DeFi",
-  "page-find-wallet-description-dcent": "D'CENT Wallet هي محفظة über المريحة متعددة العملات المشفرة مع متصفح DApp مدمج لسهولة الوصول إلى DeFi وNFT ومجموعة متنوعة من الخدمات.",
+  "page-find-wallet-description-dcent": "D'CENT Wallet is the über convenient multi-cryptocurrency wallet with built-in dapp browser for easy access to DeFi, NFT, and variety of services.",
```
Comment (minimalsm) [inline src/intl/ar/page-wallets-find-wallet.json:42]: Regression

File: src/intl/ar/page-wallets-find-wallet.json [hunk 6]
```diff
@@ -39,19 +39,19 @@
...
   "page-find-wallet-description-enjin": "غير قابل للاختراق ومليء بالميزات وملائم - مصمم للمتداولين واللاعبين والمطورين",
   "page-find-wallet-description-fortmatic": "قم بالوصول إلى تطبيقات إثيريوم من أي مكان بمجرد بريد إلكتروني أو رقم هاتف. لا مزيد من ملحقات المتصفح والعبارات الأولية.",
   "page-find-wallet-description-gnosis": "المنصة الأكثر ثقة لتخزين الأصول الرقمية على إيثيريوم",
   "page-find-wallet-description-guarda": "محفظة تشفير آمنة ومليئة بالميزات وغير مقيدة للحراسة مع دعم لأكثر من 50 بلوكشين.  الرهانات السهلة وتبادل وشراء الأصول المشفرة.",
-  "page-find-wallet-description-hyperpay": "HyperPay هي محفظة تشفير عالمية متعددة المنصات تدعم أكثر من 50 بلوكشين و 2000+ dApps.",
+  "page-find-wallet-description-hyperpay": "HyperPay is a multi-platform universal crypto wallet that supports 50+ blockchains and 2000+ dapps.",
```
Comment (minimalsm) [inline src/intl/ar/page-wallets-find-wallet.json:47]: Regression

**Comment by minimalsm** [comment]
Marking as draft due to regressions

**Review by corwintines — APPROVED** [review]

**Pull request merged**